tests/Api/TestCase.php

```diff
@@ -152,12 +152,16 @@ protected function head($uri): ResponseInterface {
     return $this->app->run(true);
   }
 
-  protected function assertContentType(ResponseInterface $response, $contentType): void {
+  protected function assertContentType($contentType, ResponseInterface $response): void {
```

Comment:
this entire time i've been reading the use of this thinking the arguments are backwards ... and now I see you switched them lol

Reply:
I didn't want to perpetuate the nonsense php does!

ie: array_filter and array_map having the callback in different spots